src/plonk.rs

```diff
@@ -74,6 +77,41 @@ impl<C: CurveAffine> VerifyingKey<C> {
             cs,
         })
     }
+
+    /// Hashes a verification key into a transcript.
+    pub fn hash_into<T: Transcript<C>>(&self, transcript: &mut T) -> io::Result<()> {
+        let mut hasher = Blake2bParams::new()
+            .hash_length(64)
+            .personal(C::BLAKE2B_PERSONALIZATION)
+            .to_state();
+
```

Comment:
Hash in the field modulus.

Reply:
I've chosen to hash in the field modulus in `domain.rs`: https://github.com/zcash/halo2/pull/173/files#diff-d87267170e09ce6a4d0d50d2af190351492d463599379ba46c6bad6b3733685bR384-R387

and the curve parameters below.